packages/mobile_app/lib/app_global.dart

```diff
@@ -30,11 +29,11 @@ class _Tester {
   bool get isNotDebug => !testerField.wrapped;
 }
 
-class StaticProvider {
+class AppGlobal {
```

Comment:
any reason of this renaming?

Reply:
I used the name StaticProvider when did not know what I created and that it was already done before me and widely used as Service Locator for Flutter Apps, especially in alpha releases around 2018.
Its correct name is App Global Singleton, and there are well-known modifications of it called GetIt https://pub.dev/packages/get_it
Since now Provider and GetIt are two main "big" competitors in Service Locators choose, I didn't want to confuse them.